src/ast/mod.rs

```diff
@@ -5867,12 +5893,12 @@ impl fmt::Display for DropFunctionOption {
 #[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
-pub struct DropFunctionDesc {
+pub struct DropFuncDesc {
```

Comment:
was it a specific reason to rename `DropFunctionDesc` to `DropFuncDesc`? if not we can probably undo the change given its a breaking change and the benefit is arguable, (and we have other similarly named objects like `DropFunctionOption`, `parse_drop_function_desc`)

Reply:
Ah yeah you're right. I playing around with some new names since DropProcedure uses it. I'll change it back.

Reply:
done in bb0dad8